JavaAdvanced_functionalProgramming/src/jv/advanced/lab/functional/programming/SumNumbers.java

```diff
@@ -0,0 +1,29 @@
+package jv.advanced.lab.functional.programming;
+
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.function.Function;
+
+/**
+ * Write a program that reads one line of Integers separated by ", ". Print the
+ * count of the numbers and their sum. Use a Function<String, Integer>
+ * 
+ * @author Ivelin Gavrailov
+ * @since 24.12.2020
+ *
+ */
+public class SumNumbers {
+
+	public static void main(String[] args) {
+		Scanner scan = new Scanner(System.in);
+		int[] numbers = Arrays.stream(scan.nextLine().split(", ")).mapToInt(Integer::parseInt).toArray();
+		Function<int[], Void> printCount = arr -> {
+			System.out.println("Count = " + arr.length);
```

Comment:
Why you did it in a different way for count and sum?

Reply:
To show that there are 2 different ways to use it.